docs/resources/vpcep_approval.md

```diff
@@ -66,14 +66,14 @@ The following arguments are supported:
 * `endpoints` (Required) - Specifies the list of VPC endpoint IDs which accepted to connect to VPC endpoint service.
     The VPC endpoints will be rejected when the resource was destroyed.
 
+* `region` - The region in which to obtain the VPC endpoint service.
+
```

Comment:
```suggestion
* `region` - (Optional, String, ForceNew) The region in which to create the resource.
  If omitted, the provider-level region will be used. Changing this creates a new resource.
```

Please move it to the first position in the parameter

Reply:
done